woocommerce/woocommerce
# Add existing global attribute layout

### All Submissions:






### Changes proposed in this Pull Request:



This PR adds a new layout to add variations for global attributes. It also allows the creation of new attributes.

When there is at least one global attribute available and no local one.

![Screen Capture on 2023-02-24 at 13-50-46](https://user-images.githubusercontent.com/1314156/221239328-e6b16138-8280-494a-b36b-601eecacfa55.gif)

Design: 6cBYbZmiet1l7gzc28ce8k-fi-5097%3A253868&t=jQo137F0guQ0Rqw0-4.

When the user selects an attribute, they land in the edit form (6cBYbZmiet1l7gzc28ce8k-fi-5096%3A253867&t=jQo137F0guQ0Rqw0-4).

Closes #36661.



### How to test the changes in this Pull Request:



1. In a brand new site. Checkout this branch
2. Go to `Products > Add New > Product data > Attributes`. You should see this form:



3. Go to `Product > Attributes` and create an attribute.
4. Go back to `Products > Add New > Product data > Attributes`.
5. The following form will be visible:

<img width="944" alt="Screenshot 2023-02-24 at 13 22 37" src="https://user-images.githubusercontent.com/1314156/221231939-8186fd98-bebd-455e-9a1c-2fd601f6a56a.png">

6. Press `Create new attribute` or press the selector and choose the attribute you just created.
7. After that the edit form should be visible:

<img width="953" alt="Screenshot 2023-02-24 at 13 25 14" src="https://user-images.githubusercontent.com/1314156/221232540-f3d2a837-2640-4b27-a25d-8588d83f49cc.png">

A follow-up PR will tackle loading the variations under `Value(s)`.

8. Remove the attribute you just added by pressing `Remove`.
9. The form we saw in step 5 will be visible again. The save button won't be visible.
10. Create a new attribute with variations and save it.



### Other information:




### FOR PR REVIEWER ONLY:
